### DISCUSSION TITLE
Added ZBL parameter to train.json in DPA2's distilltion, but ZBL in the potential function is incorrect

### DISCUSSION THREAD

**Discussion by JiangXiaoMingSan** [deepmodeling/deepmd-kit#4267] [Q&A]
I added the  ZBL parameters to the train.json  in distillation section according to the manual and successfully obtained a potential function. However, in the cascade simulation process, the number of thermal peak defects is particularly small. I used LAMMPS to calculate the energy between two atoms at a distance of 0.3, and obtained a generally small result (in my other potential functions, LAMMPS obtained a larger value than ZBL). My link range is 1.2-2, and 0.3 should be completely within the scope of ZBl. Why does it cause the values calculated by LAMMPS to not match the values in the ZBL table?

**Comment by njzjz** [comment]
Could you show the steps to reproduce this behavior?

**Reply by JiangXiaoMingSan** [reply]
Thank you for your answer. I hope you can visit my GitHub.
https://github.com/JiangXiaoMingSan/zbl_script.git

**Reply by njzjz** [reply]
There is a $E^0_i$ term in the [equation](https://docs.deepmodeling.com/projects/deepmd/en/latest/model/pairtab.html#theory) to switch smoothly.

By the way, this model is version 2 of DPA-1 rather than DPA-2.

You may consider removing your password from your public repository.

**Reply by JiangXiaoMingSan** [reply]
Thank you for your suggestion. I have changed my password and deleted it on GitHub.  In my understanding, E0 should be slightly adjusted for energy, but there is a significant difference between the two based on my calculations. For details, please refer to the zbl_vs_lammps.txt document, where the zbl value of W-W is 26444.341291428842, and the result calculated by LAMMPS is only 10056.0504641334. This results in a very small number of thermal peak defects for cascading calculations. I also tried to turn off E0 correction (by adding 'srtab_dead'bias': false) and tried to unify all images to the b3 version. However, there is a discrepancy between the calculated values obtained and ZBL.

**Reply by JiangXiaoMingSan** [reply]
The difference caused by ZBL in this potential function is huge. For example, when I use the potential function to calculate the cascade of W, there are only about 20 defects in the 5keV PKA energy thermal peak, while when I use EAM potential calculation, the number of defects in the thermal peak exceeds 200. In my other potential functions, for example, the ZBL value of V-V is 3869.2213399663797, but I calculated 9954.60341439269. At present, the potential function I constructed using DPA cannot express ZBL well.

**Reply by anyangml** [reply]
Can you provide the `lcurve` during the distillation process ?

**Reply by JiangXiaoMingSan** [reply]
Sure, I have added it to my GitHub
